source/Coop.Core/Common/Services/Connection/Handlers/PlayerConnectionServerMessageHandler.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Common.Logging;
+using Common.Messaging;
+using Common.Network;
+using Coop.Core.Client.Messages;
+using Coop.Core.Server.Connections;
+using Coop.Core.Server.Connections.Messages;
+using Coop.Core.Server.Services.Time.Messages;
+using GameInterface.Services.GameDebug.Messages;
+using LiteNetLib;
+using Serilog;
+using System.Linq;
+
+namespace Coop.Core.Common.Services.Connection.Handlers
+{
+    public class PlayerConnectionServerMessageHandler
```

Comment:
This class is never initialized, move to the server directory and have this class inherit IHandler to be collected by https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/development/source/Coop.Core/Server/ServerModule.cs#L34